policyengine_us/variables/gov/states/vt/tax/income/credits/cdcc/vt_cdcc.py

```diff
@@ -0,0 +1,19 @@
+from policyengine_us.model_api import *
+
+
+class vt_cdcc(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Vermont child care and dependent care credit"
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        "https://tax.vermont.gov/sites/tax/files/documents/IN-112-2022.pdf#page=2"
+        "https://law.justia.com/codes/vermont/2022/title-32/chapter-151/section-5828c/"
+    )
+    defined_for = StateCode.VT
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.states.vt.tax.income.credits.cdcc
+        federal_cdcc = tax_unit("cdcc", period)
```

Comment:
The [form](https://tax.vermont.gov/sites/tax/files/documents/IN-112-2022.pdf#page=2) refers to [2022 Form 2441](https://www.irs.gov/pub/irs-prior/f2441--2022.pdf) line 11, which caps the credit at tax liability.
```suggestion
        # The form refers to 2022 Form 2441 line 11, which caps the credit at tax liability.
        federal_cdcc = tax_unit("capped_cdcc", period)
```